recipes/rwa-python/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "0.8.5" %}
+
+package:
+  name: rwa-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/RWA-python/archive/v{{ version }}.tar.gz
+  sha256: 9bcc928ccfb22d1353d59a95c98062ef2b6543c56044ee92cf52ded32cb6d262
+
+build:
+  number: 0
```

This looks like it could be noarch python, if it can you should add `python >=2.7` in `host` and `run`
```suggestion
  noarch: python
  number: 0
```